code/modules/migrations/migration.dm

```diff
@@ -0,0 +1,40 @@
+/**
+ * A simple system to update the database automatically.
+ *
+ * @author Rob "N3X15" Nelson
+ */
+
+/datum/migration
```

Comment:
[Please document your code to meet dmdoc requirements.](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)